### PR TITLE
what-we-do PR ready: swiper + c-slide + c-slide-nav w/ assets

### PR DESCRIPTION
**Pull request by pananimals**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -41,6 +41,7 @@
   "dependencies": {
     "bootstrap": "4.1.x",
     "jquery": "3.3.x",
-    "popper.js": "1.14.x"
+    "popper.js": "1.14.x",
+    "swiper": "^4.4.x"
```
Comment (michalkowol) [inline package.json]: - [ ] Remove `^`

File: src/css/styles/_c-badge.scss [hunk 2]
```diff
@@ -0,0 +1,23 @@
+.c-badge {
+  display: inline-block;
+  padding: 0.2rem 0.6rem;
+  margin: 0.25rem 0;
+  margin-right: 0.5rem;
+  font-size: 1rem;
+  color: $col-light-violet;
+  background-color: $col-bg;
```
Comment (michalkowol) [inline src/css/styles/_c-badge.scss]: - [ ] Remove `background-color`

File: src/index.html [hunk 3]
```diff
@@ -76,8 +76,164 @@ <h1 class="c-banner__title">You have a concept,<br><span class="h-blue">we'll ma
...
+            </div>
+        
+            <div class="c-slide-nav">
+                <ul class="c-slide-nav__list row">
+                    <li class="c-slide-nav__item col-3">
+                        <a class="c-slide-nav__link c-slide-nav__link--active js-service" data-slide="0" role="button">
+                            <div class="c-slide-nav__icon">
+                                <img class="c-slide-nav__icon-img" src="img/apps-gray.jpg" alt="Apps">
+                            </div>
+                            <h1 class="c-slide-nav__title">App Development</h1>
```
Comment (michalkowol) [inline src/index.html]: IMHO this should not be `h1`. If we want to use `h*` maybe `h4`?

- [ ] Change `h1`

File: src/css/styles/_c-slide.scss [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+    font-size: 1.4rem;
+  }
+
+  &__description {
+    padding: 0;
+  }
+
+  &__btn {
+    padding: 0;
+    padding-right: 2rem;
```
Comment (michalkowol) [inline src/css/styles/_c-slide.scss]: - [ ] Use `padding: 0 2rem 0 0;`

File: src/css/styles/_c-slide.scss [hunk 5]
```diff
@@ -0,0 +1,52 @@
+.c-slide {
+  width: 100%;
+
+  &__row {
+    padding: 4rem 1rem 0;
+  }
+
+  &__title {
+    padding: 0;
+    padding-right: 2rem;
```
Comment (michalkowol) [inline src/css/styles/_c-slide.scss]: - [ ] Use `padding: 0 2rem 0 0;`

File: src/css/styles/_c-badge.scss [hunk 6]
```diff
@@ -0,0 +1,23 @@
+.c-badge {
+  display: inline-block;
+  padding: 0.2rem 0.6rem;
+  margin: 0.25rem 0;
+  margin-right: 0.5rem;
```
Comment (michalkowol) [inline src/css/styles/_c-badge.scss]: - [ ] Use `margin: 0.25rem 0.5rem 0.25rem 0;`

File: src/css/styles/_swiper.scss [hunk 7]
```diff
@@ -0,0 +1,7 @@
+.swiper-container {
+  margin-bottom: 2rem;
+}
+
+.swiper-scrollbar-drag {
+  background: $col-light-blue;
+}
```
Comment (michalkowol) [inline src/css/styles/_swiper.scss]: image

- [ ] 3px vs 5px

File: src/index.html (hunk 3 above)
Comment (pananimals) [reply src/index.html]: It is done for SEO purposes. H1 is analysed wrt. key phrases.

**Pull request merged**